test/sqllogictest/literal_constraints.slt

```diff
@@ -0,0 +1,405 @@
+----
+l3
+xxx
+yyy
+
+# IN list translated to ReadExistingIndex, but no semi-join or lookup, because the index is too wide
+
+query T multiline
+EXPLAIN SELECT * FROM t1
+WHERE a IN (1, 1+1)
+----
+%0 =
+| ReadExistingIndex materialize.public.idx_t1_a_b
+| Filter ((#0 = 1) OR (#0 = 2))
+
+EOF
+
+query IT rowsort
+SELECT * FROM t1
+WHERE a IN (1, 1+1)
+----
+1  a
+1  l1
+2  l2
+
+# Create the index that we would have needed in the previous query.
+
+statement ok
+CREATE INDEX idx_t1_a ON t1(a)
+
+# Same query as above, but we do a semi-join with the index. (Fast path doesn't yet handle multiple lookups.)
```

Comment:
I didn't really think much about this, because the next PR (coming today) will enable the fast path also to handle the same literal constraint situations as the slow path.